### ISSUE TITLE
redisbungee and placeholderapi issues

### ISSUE THREAD

**Issue by geumu** [minecrafter/RedisBungee#64]
I get errors when I use redisbungee.

I am using the Expansion-RedisBungee of the PlaceholderAPI and I get this error.

[05:53:12] [Server thread / ERROR]: [PlaceholderAPI] Failed to init placeholder expansion from class: com.extendedclip.expansion.redisbungee.RedisBungeeExpansion
[05:53:12] [Server thread / ERROR]: [PlaceholderAPI] Channel must contain: separator (attempted to use RedisBungee)

How can I fix this problem?

I have a 1.13.2, 1.12.2, and 1.7.10 server.

**Comment by detobel36** [comment]
This come from the new channel message.  Now we need to use `:` in channel message.        
A fix has been proposed here: #61      
But `PlaceholderAPI` need also to use this new channel format.

**Comment by geumu** [comment]
But `PlaceholderAPI` need also to use this new channel format. -> how can i do that?

**Comment by geumu** [comment]
> This come from the new channel message. Now we need to use `:` in channel message.
> A fix has been proposed here: #61
> But `PlaceholderAPI` need also to use this new channel format.

But `PlaceholderAPI` need also to use this new channel format. -> how can i do that?

how can i do that? there is only redisbungee fix information

**Comment by geumu** [comment]
> This come from the new channel message. Now we need to use `:` in channel message.
> A fix has been proposed here: #61
> But `PlaceholderAPI` need also to use this new channel format.

I can not compile. This is very painful.

I would like to run the server in 1.13.2.

May I ask you to do this?

**Comment by detobel36** [comment]
I don't know the plugin "PlaceholderAPI".    
I just tell you that the system have change.  In 1.13 the plugin which will use BungeeChannel need to have a new format with `:`.   
This format is used by Spigot plugin to communicate with Bungee plugin.  Both need to be updated.       
           
I propose a fix for RedisBungee.  This fix is not official.  We need to wait the main developer.    
But in any case, the Spigot plugin "PlaceholderAPI" need also to be update.    
     
In conclusion, for RedisBungee a fix is proposed and we waited to really implement it.  For "PlaceholderAPI" I don't see yet any issue or proposed fix.   You need to ask to the developer.

**Comment by Chnkr** [comment]
Resolved trough https://github.com/minecrafter/RedisBungee/commit/429ed39160471d0e8289ca6b662f061a08d281b2 sorry for the delay :)

**Comment by Drc-DEV** [comment]
I still get the error on 1.13.2, with RedisBungee-0.5.jar and PAPI 2.10.4

>  [16:56:00 ERROR]: [PlaceholderAPI] Failed to init placeholder expansion from class: com.extendedclip.expansion.redisbungee.RedisBungeeExpansion
[16:56:00 ERROR]: [PlaceholderAPI] Channel must contain : separator (attempted to use RedisBungee)

Merge this https://github.com/minecrafter/RedisBungee/pull/68
